addons/web/static/lib/bootstrap/js/dist/dropdown.js

```diff
@@ -665,7 +666,7 @@
 
 
   EventHandler__default.default.on(document, EVENT_KEYDOWN_DATA_API, SELECTOR_DATA_TOGGLE, Dropdown.dataApiKeydownHandler);
-  EventHandler__default.default.on(document, EVENT_KEYDOWN_DATA_API, SELECTOR_MENU, Dropdown.dataApiKeydownHandler);
+  EventHandler__default.default.on(document, EVENT_KEYDOWN_DATA_API, SELECTOR_MENU_NOT_SUB, Dropdown.dataApiKeydownHandler);
```

Comment:
Mmh, I said it was ok to change the line we already touched... :) do you actually need to differentiate `SELECTOR_MENU` with `SELECTOR_MENU_NOT_SUB`? Can't just change `SELECTOR_MENU`?

If not are you sure you can't do it outside of the lib? What prevents you to do exactly the opposite of this outside (off) then on again with your own selector?

Reply:
Doing it on all uses of `SELECTOR_MENU` makes the opening on the initial click stop working.

To be able to `.off(...)` then `.on(...)`, I need an access to the `EventHandler_default` instance. But to get access to it, I need to leak it somehow. E.g. `Dropdown.__EventHandler__default = EventHandler__default`. Just changing the selector seemed to be less intrusive.